apollo-cache-interceptor/src/commonMain/kotlin/com/apollographql/apollo3/interceptor/cache/CacheExecutionContext.kt

```diff
@@ -27,53 +20,57 @@ enum class FetchPolicy {
    *
    * This is the default behaviour
    */
-  CACHE_FIRST,
+  object CACHE_FIRST : FetchPolicy()
```

Comment:
I think we can rename it `CacheFirst` to follow Kotlin naming convention.

Reply:
Yup, 100%, will do at the same time as docs.